src/containers/PrescriptionSelectionContainer.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Dispatch } from 'redux';
+import { connect } from 'react-redux';
+import { ApplicationState } from '../store';
+import { Prescription } from '../store/order/types';
+import { Frame } from '../store/frames/types';
+import * as orderActions from '../store/order/actions';
+import { isEmptyObject } from '../helpers/mathHelper';
+import { isPrescriptionFilled } from '../helpers/orderSelectionHelper';
+import PrescriptionSelection from '../components/PrescriptionSelection';
+
+type PropsFromState = {
+  disabled: boolean,
+  errors: string,
+  prescription: Prescription,
+  frame: Frame,
+};
+
+type PropsFromDispatch = {
+  handleChange: typeof orderActions.setRxInformation,
+  savePrescription: typeof orderActions.savePrescriptionStart,
+};
+
+type ComponentProps = PropsFromState & PropsFromDispatch;
+
+type StateProps = {
+  value: any;
+};
+
+const mapStateToProps = (state: ApplicationState) => ({
```

Comment:
Perhaps you need separate file/helper object for `mapStateToProps` and `mapDispatchToProps`